docs/src/main/asciidoc/quarkus-maven-plugin.adoc

```diff
@@ -0,0 +1,40 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Quarkus Maven Plugin
+
+Builds a Quarkus application, and provides helpers to launch dev-mode, the Quarkus CLI, building of native images.
+For more information about how to use the Quarkus Maven Plugin, please refer to the xref:maven-tooling.adoc[Maven Tooling guide].
+
+include::./attributes.adoc[]
+
+== Goals
+
+Like most Maven plugins, the Quarkus Maven Plugin has a `help` goal that prints a description of the plugin, list of all available goals as well as their description.
+It's also possible to print out detail information about a goal, all its parameters and their default values. For instance, to see help for the `create` goal, type:
+
+```
+mvn quarkus:help -Ddetail -Dgoal=create
+```
+
+////
+'{empty} +' adds extra line to prevent a big gap between first goal and the table caused by long table of contents (printed by '{{ page.document | tocify_asciidoc: 2 }')
+The workaround is not responsive, but provides a better look than if it were removed.
+Once the issue is solved in the 'quarkusio.github.io' project, the workaround must be removed.
+////
+{empty} +
```

Comment:
Thank you very much. I honestly couldn't come with same meaningful text to add instead of the addition (I also didn't find out what "maven plugin targets" could be after 10 minutes of Googling). However when I removed the workaround ({empty} lines), I mentioned the gap has exactly same size as the gap here https://quarkus.io/guides/maven-tooling#project-creation (between `create command:` and the table).

The workaround is removed.

Reply:
Maven tooling page contains all important information I could think of.